### ISSUE TITLE
Remote control is needed but was removed

### ISSUE THREAD

**Issue by jhegewald** [Louisbertelsmann/Sungrow-Wallbox-Modbus-HomeAssistant#20]
Why was the remote control removed with the last merge? Isn't it needed for starting and stopping?

**Comment by SunnyCrockett** [comment]
Correct, its not needed. You start/stop charging with input buttons wb_start_charging and wb_stop_charging and check state of Charging status.

**Comment by jhegewald** [comment]
Understood - thanks a lot!



------ Originalnachricht ------
Von "SunnyCrockett" ***@***.***>
An "Louisbertelsmann/Sungrow-Wallbox-Modbus-HomeAssistant" 
***@***.***>
Cc "jhegewald" ***@***.***>; "Author" 
***@***.***>
Datum 12.07.2024 13:32:46
Betreff Re: [Louisbertelsmann/Sungrow-Wallbox-Modbus-HomeAssistant] 
Remote control is needed but was removed (Issue #20)
